contrib/packs/actions/restart_component.yaml

```diff
@@ -10,7 +10,7 @@
       immutable: true
     cmd:
       immutable: true
-      default: "st2ctl restart-component {{servicename}}"
+      default: "service {{servicename}} restart"
```

Comment:
The correct answer is to move that task in the workflow to using this action (linux.service):
https://github.com/StackStorm/st2/blob/master/contrib/linux/actions/service.py

It is currently broken but I will fix it today.